index.html

```diff
@@ -829,6 +831,70 @@ <h2><code>application/vc+ld+sd-jwt</code></h2>
       </table>
     </section>
   </section>
+  
+
+  <section id="json-web-token-claim-registrations">
+    <h3>JSON Web Token Claims</h3>
+    <p>
+      This section retained deprecated registered claim names, 
+      that have been submitted to the Internet Engineering Steering Group
+      (IESG) for review, approval, and registration with IANA in the
+      "JSON Web Token Claims Registry".
```

Comment:
Shouldn't "JSON Web Token Claims Registry" link to that resource, whether via a `[[???]]` reference or a direct `<a href...>`?
```suggestion
      This section retains deprecated registered claim names, 
      that were previously submitted to the Internet Engineering Steering
      Group (IESG) for review, approval, and registration with IANA in the
      "JSON Web Token Claims Registry".
```

Reply:
Yes, I wonder why nobody did that in VCDM?

I prefer not to make a bunch of changes here, until we finish moving the section, from there to here... but I agree that everything should be linked that can be... we should save that for editorial though.

Reply:
I merged them anyway, we will still need to wait for https://github.com/w3c/vc-data-model/pull/1195 to merge